add prefers-reduced-motion to accessibility docs (#12038)
wagtail/wagtail: docs/advanced_topics/accessibility_considerations.md

(accessibility_resources)=

### prefers-reduced-motion

> Some users may prefer to see a more static version of your site, for example, users with vestibular disorders. You can respect this preference by using the `prefers-reduced-motion` media query in your CSS.

Reviewer: A small suggestion for the sentence flow:

```suggestion
Some users, such as those with vestibular disorders, may prefer a more static version of your site. You can respect this preference by using the `prefers-reduced-motion` media query in your CSS.

```

```css
@media (prefers-reduced-motion) {
  /* styles to apply if a user's device settings are set to reduced motion */
    /* for example, disable animations */
    * {
        animation: none !important;
        transition: none !important;
    }
}
```

> Note that `prefers-reduced-motion` is only applied to users with this setting enabled in their operating system or browser. This feature is supported by Chrome, Safari and Firefox. For more information on reduced motion, see the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).

Reviewer: ```suggestion

Note that `prefers-reduced-motion` is only applied for users who enabled this setting in their operating system or browser. This feature is supported by Chrome, Safari and Firefox. For more information on reduced motion, see the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).
```
